Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <DefaultItemExcludes>$(DefaultItemExcludes);.git*;node_modules\**</DefaultItemExcludes>
+    <DefaultItemExcludes>$(DefaultItemExcludes);.*;node_modules\**</DefaultItemExcludes>
```

Comment:
Because of the addition of _.npmrc_ which triggers a ton of warnings when you open it in VS:

![2022-02-28_14h01_09](https://user-images.githubusercontent.com/164360/156066278-e5ed9564-24be-4658-99de-989388cc5b66.jpg)

Also, _.pnpm-debug.log_ appeared in the same folder, so I thought hiding all dot files would be suitable.

Reply:
I see, OK then.